internal/firehydrant/firehydrant.go

```diff
@@ -211,6 +212,8 @@ func (c *Client) CreateUser(ctx context.Context, u SCIMUser) (*store.FhUser, err
 		return nil, fmt.Errorf("creating user: %w", err)
 	}
 	if resp.StatusCode != http.StatusCreated {
+		body, _ := io.ReadAll(resp.Body)
+		console.Errorf("unexpected status code %d: %s\n", resp.StatusCode, body)
```

Comment:
log more information when response code isn't expected — this should help users get better information and can help us figure out where to look.